packages/angular/projects/dev/src/app/dropdown/dropdown-static-icon-toggle.demo.html

```diff
@@ -7,8 +7,8 @@
 <div class="clr-example squeeze demo-menu-height dropdown-demo">
   <div class="dropdown bottom-left open">
     <button class="dropdown-toggle">
-      <clr-icon shape="error" class="is-error" size="24"></clr-icon>
-      <clr-icon shape="caret down"></clr-icon>
+      <cds-icon shape="error" status="danger" size="18"></cds-icon>
```

Comment:
Is the change from size 24 to 18 expected?

Reply:
Yea, as far as I remember the visual tool mark that the `clr-icon` one was a little bit smaller than the update one so I just lower the size to satisfy the visual checker